python/pacemaker/_cts/test.py

```diff
@@ -284,7 +294,10 @@ def run_cmd(self, args):
             ### launched earlier that is listening for the failure may not have
             ### time to connect to pacemaker-execd.
             time.sleep(2)
-            subprocess.Popen(shlex.split(args['kill']))
+            # If the kill process exits with non-zero, check=False will prevent
+            # an exception from being raised.  Basically, we'll just ignore the
+            # failure.
+            subprocess.run(shlex.split(args['kill']), check=False)
```

Comment:
We're almost there but not quite :)  subprocess.run was added in Python 3.5

Reply:
Weirdly, the pylint version I have installed now does not complain about using Popen here while a previous version did.  That's why I moved to using run.  Fixed.